draft-ietf-tls-tls13.md

```diff
@@ -1056,6 +1056,12 @@ will not process the same data twice for the same connection) and
 an attacker will not be able to make 0-RTT data appear to be
 1-RTT data (because it is protected with different keys.)
 
+These properties mean that 0-RTT data must be specially treated by applications. They SHOULD be required
+to opt-in to 0-RTT data transmission or recieving it: implementations SHOULD NOT preappend recieved 0-RTT data
```

Comment:
'I' before 'E' except after 'C', so the correct spellings are "receiving" and "received".
"preappend" should be "prepend"

(also, wrap lines to around 80 characters, please)